src/renderers/shared/fiber/ReactFiberScheduler.js

```diff
@@ -1034,7 +1034,7 @@ module.exports = function<T, P, I, TI, PI, C, CX, PL>(config : HostConfig<T, P,
   function hasCapturedError(fiber : Fiber) : boolean {
     // TODO: capturedErrors should store the boundary instance, to avoid needing
     // to check the alternate.
-    return Boolean(
+    return !!(
```

Comment:
Yeah, I think it was written that way originally because `capturedErrors !== null` was just `capturedErrors`, and when it was null the expression would evaluate to null unless it was coerced. I'll change it.